e2e/Microsoft.Azure.Devices.E2ETests/MethodE2ETests.cs

```diff
@@ -333,7 +333,7 @@ public void Cleanup()
             await ServiceSendMethodAndVerifyResponse(deviceInfo.Item1, MethodName, DeviceResponseJson, ServiceRequestJson, assertResult);
 
             await deviceClient.CloseAsync();
-            TestUtil.RemoveDevice(deviceInfo.Item1, registryManager);
+            await TestUtil.RemoveDeviceAsync(deviceInfo.Item1, registryManager);
```

Comment:
Do we need ConfigureAwait()?

Reply:
No, ConfigureAwait is only needed in library code to ensure that completions are not forced to execute in the original context. 
See https://msdn.microsoft.com/en-us/magazine/jj991977.aspx for details.